project/src/main/java/com/google/sps/data/ImageLabel.java

```diff
@@ -20,7 +20,7 @@ public boolean equals(Object obj) {
     if (obj == null) return false;
     if (getClass() != obj.getClass()) return false;
     ImageLabel other = (ImageLabel) obj;
-    if (description.equals(other.getDescription())) return true;
+    if (description.toLowerCase().equals(other.getDescription().toLowerCase())) return true;
```

Comment:
Just a small stylistic choice here. You could just return the condition in this if statement. If you don't want to that's fine.